### PR TITLE
chore: update eslint deps

### PR DESCRIPTION
**Pull request by ppusapati**
# Overview

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [X ] Docs / tests / types / typos

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [X ] My code follows the [developer guidelines of this project](https://github.com/QwikDev/qwik/blob/main/CONTRIBUTING.md)
- [X ] I have performed a self-review of my own code
- [X ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Review by gioboa — CHANGES_REQUESTED** [review]
Thanks for your contribution.
eslint changed the configuration with v9
https://eslint.org/docs/latest/use/migrate-to-9.0.0
I'm not sure that this bump will work fine 🤔
Did you test this change?

**Comment by gioboa** [comment]
@ppusapati is it still valid or is it abandoned?

**Comment by ppusapati** [comment]
Yes, have tested it and its still valid. 

"@builder.io/qwik": "^1.7.3",
 "eslint-plugin-qwik": "^1.7.3",
"eslint": "^9.8.0",

File: packages/eslint-plugin-qwik/package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   "license": "MIT",
   "main": "dist/index.js",
   "peerDependencies": {
-    "eslint": "^8.57.0"
+    "eslint": "^9.7.0"
```
Comment (wmertens) [inline packages/eslint-plugin-qwik/package.json:31]: this will just force everyone to use v9 but it doesn't do anything to make us use v9

**Review by wmertens — CHANGES_REQUESTED** [review]

**Comment by wmertens** [comment]
I'm going to close this, but feel free to open a new PR with eslint updated.

**Pull request closed without merging**